### PR TITLE
Added stale issues search

### PR DESCRIPTION
**Pull request by tmasternak**
@distantcam @WojcikMike I've added the stale issue search that I've been using as a console application so far.

### CODE REVIEW

File: src/PBot/Issues/ListStaleIssues.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            "stale issues (.*)$", 
+            "pbot stale issues <tag> - lists issues which are 'State: InProgress' that look like stale")
+        {
+        }
+
+        public override async Task Execute(string[] parameters, IResponse response)
+        {
+            var client = GitHubClientBuilder.Build();
+
+            await response.Send("Looking for stale items I am ...");
```
Comment (distantcam) [inline src/PBot/Issues/ListStaleIssues.cs]: Did you mean to sound like Yoda here?

File: src/PBot/Issues/ListStaleIssues.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+                var comments = await client.Issue.Comment.GetAllForIssue("Particular", repositoryName, issue.Number);
+                var latestComment = comments.OrderByDescending(c => c.CreatedAt).FirstOrDefault();
+
+                var events = await client.Issue.Events.GetAllForIssue("Particular", repositoryName, issue.Number);
+                var movedToInProgress = events.Last(e => e.Label != null && e.Label.Name == InProgressLabel);
+
+                var stalenessTreshold = DateTime.UtcNow.Subtract(StalenessPeriod);
+
+                if (issue.CreatedAt > stalenessTreshold)
```
Comment (distantcam) [inline src/PBot/Issues/ListStaleIssues.cs]: stalenessThreshold (missing the first h in threshold)

File: src/PBot/Issues/ListStaleIssues.cs (hunk 1 above)
Comment (tmasternak) [inline src/PBot/Issues/ListStaleIssues.cs]: Yes I did :)

File: src/PBot/Issues/ListStaleIssues.cs (hunk 2 above)
Comment (tmasternak) [inline src/PBot/Issues/ListStaleIssues.cs]: :+1:

File: src/PBot/PBot.csproj [hunk 3]
```diff
@@ -20,7 +20,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\mmbot\packages\PBot.0.1.158\lib\</OutputPath>
+    <OutputPath>..\..\..\..\..\projects\mmbot\packages\PBot.0.1.160\lib\</OutputPath>
```
Comment (adamralph) [inline src/PBot/PBot.csproj]: Why has this path changed?

File: src/PBot/Issues/ListStaleIssues.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Octokit;
+
+    public class ListStaleIssues : BotCommand
+    {
+        public ListStaleIssues() : base(
+            "stale issues (.*)$", 
+            "pbot stale issues <tag> - lists issues which are 'State: InProgress' that look like stale")
```
Comment (adamralph) [inline src/PBot/Issues/ListStaleIssues.cs]: Typo: should be `'State: In Progress'`

Comment (adamralph) [inline src/PBot/Issues/ListStaleIssues.cs]: > that look like stale

sounds weird, maybe:

> that are stale

File: src/PBot/Issues/ListStaleIssues.cs (hunk 1 above)
Comment (adamralph) [inline src/PBot/Issues/ListStaleIssues.cs]: > ... stale *issues* ...

?

File: src/PBot/Issues/ListStaleIssues.cs [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+            if (searchResults.TotalCount == 0)
+            {
+                await response.Send($"No stale issues found for tag `{parameters[1]}`.");
+                return;
+            }
+
+            foreach (var issue in searchResults.Items)
+            {
+                var repositoryName = issue.CommentsUrl.AbsolutePath.Split('/')[3];
```
Comment (adamralph) [inline src/PBot/Issues/ListStaleIssues.cs]: oh dear, is Octokit that bad?

**Comment by adamralph** [comment]
@tmasternak looks good. Left a couple of minor comments.

File: src/PBot/PBot.csproj (hunk 3 above)
Comment (tmasternak) [inline src/PBot/PBot.csproj]: My bad.

File: src/PBot/Issues/ListStaleIssues.cs (hunk 4 above)
Comment (tmasternak) [inline src/PBot/Issues/ListStaleIssues.cs]: Both fixed

**Comment by adamralph** [comment]
This has been deployed in PBot 0.1.161

**Pull request merged**